common.mk

```diff
@@ -41,7 +41,7 @@ DOCKER_CLI_REF     ?= $(REF)
 DOCKER_ENGINE_REF  ?= $(REF)
 # DOCKER_COMPOSE_REF is the version of compose to package. It usually is a tag,
 # but can be a valid git reference in DOCKER_COMPOSE_REPO.
-DOCKER_COMPOSE_REF ?= v2.39.4
+DOCKER_COMPOSE_REF ?= v2.40.9
```

Comment:
`.0` probably 😅 


```
Fatal: couldn't find remote ref v2.40.9
```

Reply:
Yeah, discussion already happening on that; @vvoland where does this need to be updated in `docker/packaging` ?